### PR TITLE
Add markdown linting

### PR DESCRIPTION
**Pull request by ghost**
Solves #41

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-<a name="0.1.0"></a>
+# Changelog
```
Comment (florianrusch-cluetec) [inline CHANGELOG.md:1]: @mn-42 We should either update the changelog or get rid of it altogether.

File: README.md [hunk 2]
```diff
@@ -71,111 +81,107 @@ export class AppModule {}
...
+  every node may be draggable everywhere by default
 - `allowSelection` method that decides if node can be selected
-  `(node: NgcxTreeNodeWrapper<T>) => boolean ` - nodes are not selectable by
-  default<br><br>
+  `(node: NgcxTreeNodeWrapper<T>) => boolean` - nodes are not selectable by
+  default
 - `treeNodeContentTemplate` Angular TemplateRef that will be used to render a
-  node<br><br> `let-nodeWrapper="nodeWrapper"` may be used to access the node
+  node
+- `let-nodeWrapper="nodeWrapper"` may be used to access the node
```
Comment (florianrusch-cluetec) [inline README.md:122]: @mn-42 did I understood it correctly, that `let-nodeWrapper` should be an separate point in this list?

File: README.md [hunk 3]
```diff
@@ -71,111 +81,107 @@ export class AppModule {}
...
 
-`nodeWrapper` the input to render the node. Type: NgcxTreeNodeWrapper<T>
+`nodeWrapper`: the input to render the node. Type: [`NgcxTreeNodeWrapper<T>`](#ngcxtreenodewrappert)
 
-### Output
+#### Output
 
-`customEvent` `EventEmitter<any>` can be used to trigger the output
-'customEvent'
+`customEvent`: `EventEmitter<any>` can be used to trigger the output
```
Comment (florianrusch-cluetec) [inline README.md:178]: @mn-42 Does the `:` makes sense at these positions?

**Comment by florianrusch-cluetec** [comment]
@mn-42 Whats the purpose of the [`README-dev.md`](https://github.com/cluetec/ngcx-tree/blob/main/README-dev.md) file?

File: README.md (hunk 2 above)
Comment (mn-42) [reply README.md:122]: should be a sub-point of treeNodeContentTemplate

File: CHANGELOG.md (hunk 1 above)
Comment (mn-42) [reply CHANGELOG.md:1]: changed it... good?

**Comment by mn-42** [comment]
> @mn-42 Whats the purpose of the [`README-dev.md`](https://github.com/cluetec/ngcx-tree/blob/main/README-dev.md) file?

information for developers, that may help to understand the code

**Comment by mn-42** [comment]
closed due to inactivity

**Pull request closed without merging**